### PR TITLE
hw-mgmt: scripts: Add missed asic thermal attributes for multi-asic systems

### PR DESCRIPTION
**Pull request by sholeksandr**
Add missed asic thermal attributes for multi-asic systems.
Added:

$tpath/asic{N}_temp_trip_crit
$tpath/asic{N}_temp_emergency
$tpath/asic{N}_temp_crit
$tpath/asic{N}_temp_norm

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

**Review by ciju-nvidia — APPROVED** [review]

File: usr/usr/bin/hw-management-thermal-events.sh [hunk 1]
```diff
@@ -336,20 +336,29 @@ if [ "$1" == "add" ]; then
 						asic_index=${asic_indices[${asic_bus}]}
 						ln -fs "$3""$4" "$cpath"/asic${asic_index}_hwmon
 						check_n_link "$3""$4"/temp1_input "$tpath"/asic${asic_index}
+						echo 120000 > $tpath/asic${asic_index}_temp_trip_crit
```
Comment (acoifmannvidia) [inline usr/usr/bin/hw-management-thermal-events.sh:339]: can we use constants as readonly or declare -r?

**Comment by acoifmannvidia** [comment]
hi, how are we testing it?

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**